qt_ui/windows/newgame/WizardPages/QGeneratorSettings.py

```diff
@@ -154,34 +154,34 @@ def __init__(self, campaign: Campaign, parent=None):
         modLayout_row = 1
 
         mod_pairs = [
-            ("F9F Panther (v2.8.7.101)", self.f9f_panther),
             ("A-4E Skyhawk (v2.2.0)", self.a4_skyhawk),
             ("A-6A Intruder (v2.7.5.01)", self.a6a_intruder),
             ("A-7E Corsair II", self.a7e_corsair2),
-            ("C-130J-30 Super Hercules", self.hercules),
-            ("F-4B/C Phantom II (2.8.7.204)", self.f4bc_phantom),
+            ("C-130J-30 Super Hercules (v6.8.2)", self.hercules),
+            ("F-100 Super Sabre (v2.7.18.30765 patch 20.10.22)", self.f100_supersabre),
+            ("F-104 Starfighter (v2.7.11.222.01)", self.f104_starfighter),
+            ("F-105 Thunderchief (v2.7.12.23x)", self.f105_thunderchief),
             ("F-15D Baz (v1.0)", self.f15d_baz),
             ("F-15I Ra'am (v1.0 by IDF Mods Project)", self.f_15_idf),
             ("F-16I Sufa & F-16D (v3.6 by IDF Mods Project)", self.f_16_idf),
-            ("F/A-18E/F/G Super Hornet (version 2.2.5)", self.fa_18efg),
-            ("F/A-18E/F Super Hornet AI Tanker (version 1.4)", self.fa18ef_tanker),
             ("F-22A Raptor", self.f22_raptor),
+            ("F-4B/C Phantom II (2.8.7.204)", self.f4bc_phantom),
             ("F-84G Thunderjet (v2.5.7.01)", self.f84g_thunderjet),
-            ("F-100 Super Sabre (v2.7.18.30765 patch 20.10.22)", self.f100_supersabre),
-            ("F-104 Starfighter (v2.7.11.222.01)", self.f104_starfighter),
-            ("F-105 Thunderchief (v2.7.12.23x)", self.f105_thunderchief),
-            ("Frenchpack", self.frenchpack),
+            ("F9F Panther (v2.8.7.101)", self.f9f_panther),
+            ("F/A-18E/F Super Hornet AI Tanker (version 1.4)", self.fa18ef_tanker),
+            ("F/A-18E/F/G Super Hornet (version 2.2.5)", self.fa_18efg),
+            ("Frenchpack (v4.9.1)", self.frenchpack),
```

Comment:
I'm not entirely sure if we support v4.9.1, but nevertheless I'll go ahead & merge... If it turns out we're not supporting a particular unit for the latest version, we'll handle it then...